src/bootstrap/test.rs

```diff
@@ -229,6 +229,9 @@ impl Step for Cargo {
         cargo.env("CFG_DISABLE_CROSS_TESTS", "1");
         // Disable a test that has issues with mingw.
         cargo.env("CARGO_TEST_DISABLE_GIT_CLI", "1");
+        // Forcibly disable tests using nightly features since any changes to
+        // those features won't be able to land.
+        cargo.env("CARGO_TEST_DIABLE_NIGHTLY", "1");
```

Comment:
I think this is a typo: `CARGO_TEST_DIABLE_NIGHTLY` should be `CARGO_TEST_DISABLE_NIGHTLY` (missing 's' in "disable").